Makefile

```diff
@@ -71,6 +75,7 @@ ifeq ("Darwin", "$(shell uname -s)")
 else
 	cd viam-cartographer/scripts && ./setup_cartographer_linux.sh
 endif
+	@make artifact-pull
```

Comment:
Added so that when we run `make setup` within CI & on local machines, we automatically download all the artifact data needed to run tests.